src/drivers/SpiNorFlash.cpp

```diff
@@ -23,10 +24,12 @@ void SpiNorFlash::Uninit() {
 void SpiNorFlash::Sleep() {
   auto cmd = static_cast<uint8_t>(Commands::DeepPowerDown);
   spi.Write(&cmd, sizeof(uint8_t));
+  spi.Sleep();
```

Comment:
Since "sleep" in this instance only means changing the pin configuration, can this be done automatically inside the read and write functions?

Reply:
See my other comment, calling Sleep here might not be a good idea. I'll do the change :)